docs/templates/resource_model.md.j2

```diff
@@ -0,0 +1,20 @@
+{% if model.description %}
+## Description
+{{ model.description | escape_md_symbols }}
+{% endif %}
+{% if model.operations %}
+
+## Supported Operations
+{% for operation in model.operations %}
+* [{{ operation["name"] }}](../resources/{{ operation["tag"] }}/{{ operation["name"] | camel_to_snake }}.md)
+{% endfor %}
+{% endif %}
+{% if model.properties %}
+
+## Model Properties
```

Comment:
Model Properties can be extracted in a separate "include" template, they are also used in `model.md.j2` template.

Reply:
done